rspace/src/test/scala/coop/rchain/rspace/history/HistoryActionsTests.scala

```diff
@@ -398,18 +399,18 @@ abstract class HistoryActionsTests[T] extends HistoryTestsBase[T, TestKey, ByteV
       insert(store, key5, val5)
       insert(store, key6, val6)
 
-      val leaves = getLeaves(store)
+      val leaves = getRoot(store).map(getLeaves(store, _))
 
-      leaves should contain allElementsOf expected
+      leaves.value should contain allElementsOf expected
```

Comment:
This assertion will succeed even if `leaves.value` contains some superfluous elements. Better use `theSameElementsAs`